### ISSUE TITLE
Slow `openFile` request may disrupt initial loading of a project

### ISSUE THREAD

**Issue by hubertp** [enso-org/enso#11721]
Users reported no/very slow loading of some suggestions. The issue is particularly on slow/underpowered machines.
While lack of suggestions is the end result, the problem appears in ydoc not dealing well with receiving answers to its request well beyond its hardcoded limit (15secs). 

This is not a new issue, and mitigation has been added to [backend/LS](https://github.com/enso-org/enso/pull/10549) as well as GUI. It appears that the mitigation has not been sufficient/loading times have deteriorated.

**Comment by hubertp** [comment]
Trying to optimize startup a bit where Akka has a big share of the pie and stumbled upon https://github.com/akka/akka/issues/25569. 

TL; DR 
For a cold JVM, there is little to nothing we can do apart from using a native image.

![Screenshot from 2024-12-02 13-09-48](https://github.com/user-attachments/assets/990c62bc-ff29-4d8a-ad99-e3e46c58daa9)

**Comment by hubertp** [comment]
Given that opt-in native-image helped tremendously with initial startup I'm tempted to close this one once native-image becomes a default on cloud.

**Comment by hubertp** [comment]
No longer seeing problems in the cloud. Closing.
